code/clue.js

```diff
@@ -1,26 +1,131 @@
-// CREATE OBJECTS FOR ALL THE SUSPECTS, SOMETHING LIKE THIS:
-
+// OBJECTS FOR ALL THE SUSPECTS
 const mrGreen = {
   firstName: "Jacob",
   lastName: "Green",
-  color: "green",
+  color: "rgba(48, 75, 59, 0.8)",
   description: "He has a lot of connections",
-  age: 45,
-  image: "assets/green.png",
+  age: "Age: 45",
```

Comment:
As of age being a number you can add this value as a int instead of a string. The difference is that a string is for example a word and needs `''` or `""` around it. An int doesn't need `''` or `""` around it - it can be clear numbers. So in this example you can type it as `age: 45`